.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,29 @@
+name: spellcheck
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: check out code
+      uses: actions/checkout@v2
+
+    - name: install misspell
+      run: |
+        curl -L -o ./install-misspell.sh https://git.io/misspell
+        sh ./install-misspell.sh
+
+    - name: run misspell
+      run: |
+        ./bin/misspell -error ./**/*.h
+        ./bin/misspell -error ./**/*.cc
+        ./bin/misspell -error ./**/*.cmd
+        ./bin/misspell -error ./**/*.md
+        ./bin/misspell -error ./**/*.sh
+        ./bin/misspell -error ./**/*.txt
```

Comment:
This covers CMakeLists.txt. Should we also cover BUILD files?

Reply:
Sure. Do you think we should just run against `*` (so we don't have to maintain a list here)?

Reply:
Dunno. :) Your call.